floor_generator/lib/enum_helper_generator.dart

```diff
@@ -0,0 +1,87 @@
+import 'dart:async';
```

Comment:
❓ What functionality does this class provide?

Reply:
This code generates an extension for the enuns ones to get the value and description that is provided in the annotations, example below.


### enum

```DART
enum CafeAspecto {
  @Description('Bom')
  @JsonValue(1)
  bom,
  @Description('Regular')    // description to show to user
  @JsonValue(2)                  // value to save database
  regular,
  @Description('Mau')
  @JsonValue(3)
  mau,
}
```

### generated extension

```DART
extension CafeAspectoValueExtension on CafeAspecto {
  int get value {
    switch (this) {
      case CafeAspecto.bom:
        return 1;
      case CafeAspecto.regular:
        return 2;
      case CafeAspecto.mau:
        return 3;
    }
  }

  String get description {
    switch (this) {
      case CafeAspecto.bom:
        return 'Bom';
      case CafeAspecto.regular:
        return 'Regular';
      case CafeAspecto.mau:
        return 'Mau';
    }
  }
}
```

### Example usage

```DART
final value = CafeAspecto.bom.value;              // value is 1
final description = CafeAspecto.bom.description;  //description is Bom
```

I intend to put it to get this value and save it in the database automatically instead of having a type converter for each enumerated